workshop/content/docs/api-deep-dive/tiles.md

```diff
@@ -566,6 +566,78 @@ Map tile:
 
 You can see [here](https://maps.gnosis.earth/ogcapi/collections/blueMarble/map/tiles?f=json) an example of a request for a (map) tileset list and [here](https://maps.gnosis.earth/ogcapi/collections/blueMarble/map/tiles/GoogleCRS84Quad?f=json) an example of a request for (map) tileset metadata.
 
+### Client usage
+
+Various software supports the OGC API - Tiles standard (see <https://github.com/opengeospatial/ogcapi-tiles/blob/master/implementations.adoc#clients> for a list of supporting implementations).
+
+In this section we will demonstrate examples using OpenLayers, QGIS, and pygeoap.
+
+#### OpenLayers 
+
+[OpenLayers](https://openlayers.org) started supporting OGC Vector tiles from V.7. by introducing OGCVectorTile Source which can be used in the VectorTile layer type.
+
+An example of this can be seen on the [example page on the OpenLayers website](https://openlayers.org/en/latest/examples/ogc-vector-tiles.html).
+
+```javascript
+import MVT from 'ol/format/MVT.js';
+import Map from 'ol/Map.js';
+import OGCVectorTile from 'ol/source/OGCVectorTile.js';
+import VectorTileLayer from 'ol/layer/VectorTile.js';
+import View from 'ol/View.js';
+
+const map = new Map({
+  target: 'map',
+  layers: [
+    new VectorTileLayer({
+      source: new OGCVectorTile({
+        url: 'https://demo.ldproxy.net/zoomstack/tiles/WebMercatorQuad',
+        format: new MVT(),
+      }),
+      background: '#d1d1d1',
+      style: {
+        'stroke-width': 0.6,
+        'stroke-color': '#8c8b8b',
+        'fill-color': '#f7f7e9',
+      },
+    }),
+  ],
+  view: new View({
+    center: [0, 0],
+    zoom: 1,
+  }),
+});
+
+```
+![vector_tiles](../assets/images/vector_tiles.png){width="100.0%"}
+
+#### QGIS
+
+Recent Verion of QGIS supports adding OGC API - Tiles under adding `new raster data`.
+
+![qgis_tiles](../assets/images/qgis_tiles.png){width="100.0%"}
+
+#### pygeoapi 
```

Comment:
@tomkralidis @krishnaglodha  I don't think we should have pygeoapi in the client usage of tiles. It may just confuse people :) We'll have plenty of time to cover publishing tiles using pygeoapi in the diving into pygeoapi workshop.